bindata/etcd/cluster-restore.sh

```diff
@@ -63,7 +63,17 @@ function extract_and_start_restore_etcd_pod() {
    fi
 
    echo "starting restored etcd-pod.yaml"
-   tar -O -xvf "${BACKUP_FILE}" -C "${MANIFEST_DIR}"/ "${BACKUP_POD_PATH}" > "${MANIFEST_DIR}"/etcd-pod.yaml
+   tar -O -xvf "${BACKUP_FILE}" -C "${MANIFEST_DIR}"/ "${BACKUP_POD_PATH}" | \
+     sed -e 's,export ETCD_NAME=.*$,export ETCD_NAME=${NODE_NODE_ENVVAR_NAME_ETCD_NAME},g' \
```

Comment:
It makes me nervous that we have no error handling with this. Should we be concerned?

Reply:
If there is an error, the script will fail.

Reply:
if the replacement does not happen would we know?

Reply:
my point is we are assuming sed replacement is a success, is that a concern.

Reply:
@hexfusion updated with a function which checks all patterns first before replacing them.